### PR TITLE
Keep score and show score on client

### PR DESCRIPTION
**Pull request by elijahkim**

### CODE REVIEW

**Comment by elijahkim** [comment]
handles https://github.com/elijahkim/reactor/issues/23 and https://github.com/elijahkim/reactor/issues/29

File: web/static/js/game.js [hunk 1]
```diff
@@ -60,11 +60,13 @@ class Game extends Component {
 
     this.channel.on("new:winner", (msg) => {
       const { messages } = this.state;
+      console.log(msg);
```
Comment (cjbell) [inline web/static/js/game.js]: remove

File: web/channels/game_channel.ex [hunk 2]
```diff
@@ -8,7 +8,8 @@ defmodule Reactor.GameChannel do
   end
 
   def broadcast_winner(topic, %{user: user}) do
-    Reactor.Endpoint.broadcast("game:#{topic}", "new:winner", %{user: user})
+    {:ok, users} = GameManager.get_users(topic)
```
Comment (cjbell) [inline web/channels/game_channel.ex]: `topic` should probably be `game_id`

File: lib/game/game.ex [hunk 3]
```diff
@@ -66,4 +66,11 @@ defmodule Reactor.Game do
 
     {:noreply, state}
   end
+
+  def handle_cast({:handle_winner, %{winner: winner}}, state) do
+    {_, state} = get_and_update_in(state, [:users, winner, :score], &{&1, &1 + 1})
```
Comment (cjbell) [inline lib/game/game.ex:71]: fancy

File: web/channels/game_channel.ex (hunk 2 above)
Comment (elijahkim) [inline web/channels/game_channel.ex]: I'm not sure what this should be called. In Channel terms they call it a topic but it's also the game_id

File: lib/game/game.ex (hunk 3 above)
Comment (elijahkim) [inline lib/game/game.ex:71]: copy pasta!

File: web/channels/game_channel.ex (hunk 2 above)
Comment (cjbell) [inline web/channels/game_channel.ex]: i'd go for what makes sense in your nomenclature here, which is `game_id`

**Pull request merged**